padd.sh

```diff
@@ -490,7 +490,7 @@ GetVersionInformation() {
     fi
 
     # PADD version information...
-    padd_version_latest=$(curl -sI https://github.com/jpmck/PADD/releases/latest | grep -i 'Location' | awk -F '/' '{print $NF}' | tr -d '\r\n[:alpha:]')
+    padd_version_latest=$(curl -sI https://github.com/jpmck/PADD/releases/latest | grep -i '^Location:' | awk -F '/' '{print $NF}' | tr -d '\r\n[:alpha:]')
```

Comment:
```suggestion
    padd_version_latest=$(curl -sI https://github.com/jpmck/PADD/releases/latest | grep -i '^Location: ' | awk -F '/' '{print $NF}' | tr -d '\r\n[:alpha:]')
```

Reply:
HTTP RFC does not require the presence of space, therefore not having the space is more robust / future proof.

please see 
https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4
> The field value MAY be preceded by any amount of LWS, though a single SP is preferred.

and 
https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

> The field-content does not include any leading or trailing LWS: linear white space occurring before the first non-whitespace character of the field-value or after the last non-whitespace character of the field-value. Such leading or trailing LWS MAY be removed without changing the semantics of the field value. Any LWS that occurs between field-content MAY be replaced with a single SP before interpreting the field value or forwarding the message downstream.